ruby.rb

```diff
@@ -7,4 +7,47 @@
 # Add comments to your code explaining your approach
 
 class Bike
-end
\ No newline at end of file
+  attr_accessor :model, :wheels, :current_speed
+  def initialize(model)
+    @model = model
+    @wheels = 2
+    @current_speed = 0
+  end
+
+  # Method that takes number as parameter and adds it to current speed which will always start with 0
+  def pedal_faster(number)
+    @current_speed += number
+  end 
+  
+  # Method that takes a number as parameter and adds to the wheels. but you cant exceed more that 3 since we only make tricycles! 
+  def tricycle(number)
+    if @wheels + number > 3
+      p "we only make tricycles!"
```

Comment:
This method looks great but I would make sure the `p` is on the method call down below on line 50 and not on the inner workings of your code.